### DISCUSSION TITLE
darkmode: code shown white on light green on readthedocs

### DISCUSSION THREAD

**Discussion by tdegeus** [pradyunsg/furo#509] [Potential Issue] [answered]
I have my docs setup with (nothing special)
```python
project = "GooseSLURM"
copyright = "2018-2022, Tom de Geus"
author = "Tom de Geus"
extensions = [
    "sphinx.ext.imgmath",
    "sphinx.ext.autodoc",
    "sphinx.ext.autosummary",
    "sphinxarg.ext",
]
exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
html_theme = "furo"
```
And some part of my docs are
```rst
******************
Getting GooseSLURM
******************

Using conda
===========

The easiest is to use *conda* to install *GooseSLURM*::

    conda install -c conda-forge gooseslurm

This will install all the necessary runtime dependencies as well.
```
They render beautiful on my system. But built on readthedocs with (nothing special)
```python
sphinx:
  configuration: docs/conf.py

version: 2

conda:
  environment: environment.yaml

python:
  install:
  - method: pip
    path: .
  system_packages: true
```
renders code as white on light green on https://gooseslurm.readthedocs.io/en/latest/getting.html# :  
<img width="768" alt="dark" src="https://user-images.githubusercontent.com/7030445/186954105-ed2041d8-60eb-4fd6-9699-af91ca5e5c15.png">

Strongly I did not find other issues with this...?

> I don't have this issue with all of my repos it seems

**Comment by pradyunsg** [comment, marked as answer]
Do a hard-refresh? (shift+F5)

This might be a case of having stale cached styles, in the local browser.

**Reply by tdegeus** [reply]
Thanks. That was it! Sorry for the noise
